### ISSUE TITLE
Hypernetworks Not Having any Effect on Prompts

### ISSUE THREAD

**Issue by AIShank** [TheLastBen/fast-stable-diffusion#158]
OS: Windows
Browser: Firefox/Chrome (Tried in both)

I noticed at least one other person here having a similar issue, but I figured I could at least post a bit more context. Simply put, hypernetworks seem to be broken currently. I haven't messed with hypernetworks until today, so I'm unsure if this is a recent issue or not. The hypernetwork files are in the right folder, and appear in the drop down menu in settings, hypernetwork strength is also set to 1. I update the repo each time I use the AI, so this should be the up to date version.

![chrome_F1qR5Yjf9m](https://user-images.githubusercontent.com/115922400/196028547-abb90157-a24d-4fcc-988f-9ee752b713af.png)

![chrome_XEEwZeHmrr](https://user-images.githubusercontent.com/115922400/196028366-a3eb7974-b3ab-4eaa-8cfb-ecac6a7a5507.png)

The console also appears to be loading them just fine, as you can see here when I tried to generate a grid comparing the different networks I tried.

![chrome_ex6Ec3nVbl](https://user-images.githubusercontent.com/115922400/196028402-33969672-4971-4395-8857-067854e7f238.png)

Here is the example output I tested, no changes in between the images at all.

![chrome_vibuZMJheP](https://user-images.githubusercontent.com/115922400/196028423-8c9332c2-40af-46b7-a24c-d25e0df89254.png)

Not sure if there is anything on your end you can do, or if this is an issue to take up with Voldy, but I only use the colab so I figured I'd post this here first, thanks in advance.

**Comment by TheLastBen** [comment]
Thanks for the detailed explanation, try experimenting with **--deepdanbooru** argument and **--hypernetwork-dir** and **--embeddings-dir**
if you get any results that would be of great help.

**Comment by Inkorak** [comment]
I have a suspicion that the problem is in the xformers, because hypernetworks affect cross attention. I can't right now, but can anyone try running a colab and hypernetworks without xformers?

**Comment by TheLastBen** [comment]
if you want to run it without xformers, just comment out the line :
`!wget -O attention.py https://raw.githubusercontent.com/TheLastBen/fast-stable-diffusion/main/precompiled/attention.py `
in the requirements cell

**Comment by AIShank** [comment]
> Thanks for the detailed explanation, try experimenting with **--deepdanbooru** argument and **--hypernetwork-dir** and **--embeddings-dir** if you get any results that would be of great help.

Sorry I have 0 experience coding and haven't ever looked at the code for the Colab, could you send a screenshot of where I would input this code?

**Comment by TheLastBen** [comment]
@AIShank I have updated the notebook, try loading the latest colab from the main page and try if the hypernetwork works

**Comment by AIShank** [comment]
> @AIShank I have updated the notebook, try loading the latest colab from the main page and try if the hypernetwork works

Seems like I'm still getting the same results, maybe I could try those suggestions from before, if you could show me where to input that code.

![index](https://user-images.githubusercontent.com/115922400/196053917-ff32ff50-2ea9-42be-8f4d-d8b410b52ba3.png)

**Comment by TheLastBen** [comment]
I don't think commenting that line will fix the problem with hypernetwork, I'm still looking in the AUTOMATIC1111 repo for answers

**Comment by AIShank** [comment]
> Thanks for the detailed explanation, try experimenting with **--deepdanbooru** argument and **--hypernetwork-dir** and **--embeddings-dir** if you get any results that would be of great help.

Not the xformers one, but these? Or did you mean both of them wouldn't help.

**Comment by AIShank** [comment]
I added the --deepdanbooru argument like in your screenshot; these are my interrogate settings, do any of these need to be changed?

![firefox_DQAWa9ToVd](https://user-images.githubusercontent.com/115922400/196054598-317d0876-b3f5-4ecb-8087-9ec57099b4b1.png)

If not, my tests are still not changing when using hypernetworks.

**Comment by TheLastBen** [comment]
We're gonna have to wait for more feedback

**Comment by AIShank** [comment]
Fair enough, thanks for the quick responses. Hope we can figure out the issue.

**Comment by Inkorak** [comment]
@TheLastBen ok, with Hypernetwork Compatibility it's works now.
![xy_grid-0000-2870305590](https://user-images.githubusercontent.com/52286717/196074365-f635d5cb-c4bc-4787-9880-99497c525922.png)
